webpack/createConfig.js

```diff
@@ -7,6 +7,7 @@ function createConfig(params) {
   const WebpackNotifierPlugin = require("webpack-notifier");
   const ParallelUglifyESPlugin = require("webpack-parallel-uglify-es-plugin");
   const WebpackAssetsManifest = require("webpack-assets-manifest");
+  // const bourbon = require("bourbon");
```

Comment:
A few comments in here are a note about how we'd add bourbon, if we wanted it. But I don't think we actually need it, so maybe we should just delete the comments.